src/lib.js

```diff
@@ -16,7 +16,11 @@ import SearchField from "./components/searchField/SearchField";
 import TriggerField from "./components/triggerField/TriggerField";
 
 import build from "./util/DebugIDUtil";
-import { formatDate, getCurrentDateFormatted } from "./util/DateFormatter";
+import {
+    formatDate,
+    formatCurrentDate,
+    formatDateObject,
```

Comment:
This `formatDateObject` function is now getting imported, but it's not exported below.
I'd expect this to cause some kind of `unused import` warning from ESLint.